### PR TITLE
Add interview preparation

### PR DESCRIPTION
**Pull request by ajmompr**
Hi @heratyian! @bchandhu and I have gone through and updated the 2024 intro curriculum repo. We have added the interview prep module & removed the web content and updated the intro curriculum modules to match the new formatting! May you please review?

### CODE REVIEW

File: cohorts/we-2024-7-1-intro/preparation-for-interview.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Preparation for Interview
```
Comment (heratyian) [inline cohorts/we-2024-7-1-intro/preparation-for-interview.md:1]: I think we can refactor this module a bit. The 2 big points to prepare for (maybe there's more) are technical interview and panel interview. I think we can have 1 assignment for each (technical, panel) with all the tips, tricks, and links to the lessons they should review.

For technical it's probably exercism, the 2 class lessons, debugging, using vscode, etc.

File: cohorts/we-2024-7-1-intro/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+![](./assets/welcome-page-banner.gif)
+
+This "tech prep" curriculum is designed to be remote, self-paced, and prepare you for success in the 12-week in person full stack web development apprenticeship program.
+Learning to code is an exponential process that requires innate interest, grit and a lifelong commitment to learning.
+
+[Welcome Video](./assets/intro-welcome.mp4)
+
+## Modules
+- [Getting Started](./getting-started.md)
+- [Career Development](./career-development.md)
```
[the remark this answers is not in the document]
Comment (ajmompr) [reply cohorts/we-2024-7-1-intro/index.md:11]: Sent out an email requesting her feedback/OK!

File: cohorts/we-2024-7-1-intro/preparation-for-interview.md (hunk 1 above)
Comment (ajmompr) [reply cohorts/we-2024-7-1-intro/preparation-for-interview.md:1]: @bchandhu and I met this afternoon, and we will be working on splitting up the content with a technical & panel interview prep section.

File: cohorts/we-2024-7-1-intro/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajmompr) [reply cohorts/we-2024-7-1-intro/index.md:11]: @bchandhu I copied you onto the email with Morgan. She gave us the OK to merge into one module on 6/28! This is the request I was referring to on github :)

**Comment by ajmompr** [comment]
Hi @heratyian! @bchandhu and I have gone through and made the requested edits. May you please review?

**Review by heratyian — APPROVED** [review]

**Pull request merged**